docs/contracts/visibility-and-getters.rst

```diff
@@ -221,4 +221,4 @@ to select individual struct members or provide a key for the mapping:
         a = data[arg1][arg2][arg3].a;
         b = data[arg1][arg2][arg3].b;
         e = data[arg1][arg2][arg3].e;
-    }
+    }
```

Comment:
Any clue why this is showing up in the diff?

Reply:
No actually. There is no trailing whitespaces and these two symbols are the same. probably a github bug?

Reply:
Yeah, probably a bug, Just asking out of curiosity.